pkg/sfu/connectionquality/connectionstats.go

```diff
@@ -97,72 +151,93 @@ func (cs *ConnectionStats) updateScore(streams map[uint32]*buffer.StreamStatsWit
 	}
 
 	cs.lastUpdate = time.Now()
-	maxAvailableLayer, maxAvailableLayerStats := cs.getMaxAvailableLayerStats(streams)
-	if maxAvailableLayerStats == nil {
-		// retain old score as stats will not be available when muted
-		return cs.score
-	}
-
-	params := TrackScoreParams{
-		Duration:        maxAvailableLayerStats.Duration,
-		Codec:           cs.params.CodecName,
-		PacketsExpected: maxAvailableLayerStats.Packets + maxAvailableLayerStats.PacketsPadding,
-		PacketsLost:     maxAvailableLayerStats.PacketsLost,
-		Bytes:           maxAvailableLayerStats.Bytes - maxAvailableLayerStats.HeaderBytes, // only use media payload size
-		Frames:          maxAvailableLayerStats.Frames,
-		Jitter:          maxAvailableLayerStats.JitterMax,
-		Rtt:             maxAvailableLayerStats.RttMax,
-	}
 
 	switch {
-	case cs.trackSource == livekit.TrackSource_SCREEN_SHARE:
-		if cs.params.GetIsReducedQuality != nil {
-			params.IsReducedQuality = cs.params.GetIsReducedQuality()
+	case cs.trackInfo.Type == livekit.TrackType_AUDIO:
+		maxAvailableLayer, maxAvailableLayerStats := getMaxAvailableLayerStats(streams, buffer.InvalidLayerSpatial)
+		if maxAvailableLayerStats == nil {
+			// retain old score as stats will not be available when muted
+			return cs.score
 		}
-		cs.score = ScreenshareTrackScore(params)
 
-	case cs.params.CodecType == webrtc.RTPCodecTypeAudio:
+		params := getTrackScoreParams(cs.codecName, maxAvailableLayerStats)
 		packetRate := float64(params.PacketsExpected) / maxAvailableLayerStats.Duration.Seconds()
-		if packetRate > audioPacketRateThreshold {
+		if packetRate < audioPacketRateThreshold {
 			// With DTX, it is possible to have fewer packets per second.
 			// A loss with reduced packet rate has amplified negative effect on quality.
 			// Opus uses 20 ms packetisation (50 pps). Calculate score only if packet rate is at least half of that.
-			if cs.params.IsDtxDisabled != nil {
-				params.DtxDisabled = cs.params.IsDtxDisabled()
-			}
```

Comment:
Removing this (i. e. leaving DTX = false which is default). Reason being rtc score treats the entire window is DTX if we say DTX = true. As we have a packet threshold check here, not setting DTX enabled.